docs/gno-tooling/cli/gnodev.md

```diff
@@ -35,15 +36,45 @@ For hot reloading, `gnodev` watches the examples folder, as well as any specifie
 gnodev ./myrealm
 ```
 
+## Keybase and Balance
+
+Gnodev will, by default, load your keybase located in the GNOHOME directory, giving all your keys (almost) unlimited fund. 
+
+All realms will be by the `-genesis-creator`, but You can also pass query options to the realm path to load a
+realm with specific creator and deposit:
+```
+gnodev ./myrealm?creator=foo&deposit=42ugnot``
+```
+
+### Additional Account
+Use the `-add-account` flag with the format `<bech32/name>[:<amount>]` to add a specific address or key name
+from your local keybase. You can set an optional amount for this address. Repeat this command to add multiple
+accounts. The addresses will be shown during runtime or by pressing `A` to display accounts interactively.
+
+## Interactive Usage
+
 While `gnodev` is running, the following shortcuts are available:
+- To see help, press `H`.
+- To display accounts balances, press `A`.
 - To reload manually, press `R`.
 - To reset the state of the node, press `CMD+R`.
-- To see help, press `H`.
 - To stop `gnodev`, press `CMD+C`.
 
 ### Options
 
-| Flag       | Effect                                              |
-|------------|-----------------------------------------------------|
-| --minimal  | Start `gnodev` without loading the examples folder. |
-| --no-watch | Disable hot reload.                                 |
\ No newline at end of file
+| Flag                | Effect                                                  |
+|---------------------|---------------------------------------------------------|
+| --minimal           | Start `gnodev` without loading the examples folder.     |
+| --no-watch          | Disable hot reload.                                     |
+| --add-account       | Pre-add account(s) in the form `<bech32>[:<amount>]`    |
+| --balances-file     | Load a balance for the user(s) from a balance file.     |
+| --chain-id          | Set node ChainID                                        |
+| --genesis-creator   | Name or bech32 address of the genesis creator           |
```

Comment:
Nice, much better! I was looking for this one, haha! What do you think about simplify this even more using `--deploy-key`? I say this because I now realize that the term `deploy` fits well, but not so much for `genesis`. This specific key is used in `AddMsg` if the `creator` hasn't been specified by the user to upload a package:

- **--deploy-key**: Default key name or Bech32 address for uploading packages.

Reply:
This also means renaming the term `creator` to `deployer`

Reply:
Yeah, actually `-deploy-key` might be the best option